apps/web/src/app/api/v1/workspaces/[wsId]/promotions/route.ts

```
@@ -0,0 +1,33 @@
+
+export async function POST(req: Request, { params }: Params) {
+  const supabase = await createClient();
+  const data = await req.json();
+  const { wsId } = await params;
+
+  const { error } = await supabase.from('workspace_promotions').insert({
+    ...data,
+    ws_id: wsId,
+    // TODO: better handling boolean value, as expand to further units
+    unit: undefined,
+    use_ratio: data.unit === 'percentage' ? true : false,
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)